mods/ra2/mod.yaml

```diff
@@ -88,7 +89,6 @@ Sequences:
 	ra2|sequences/civilian-props.yaml
 	ra2|sequences/civilian-structures.yaml
 	ra2|sequences/defaults.yaml
-	ra2|sequences/misc.yaml
```

Comment:
With the move of the resource sequences inside the file, is moving the file reference still needed?

Reply:
The whole issue seemed to be resolved upstream since 2 years ago.